InnerEye/ML/model_training.py

```diff
@@ -290,3 +280,24 @@ def model_train(config: ModelConfigBase,
         resource_monitor.kill()
 
     return model_training_results
+
+
+def aggregate_and_create_subject_metrics_file(config: ModelConfigBase) -> None:
```

Comment:
The signature could be simplified (and hence testing made easier) if it only consumed an outputs_folder argument.